scalafix-tests/input/src/main/scala/test/ExplicitReturnTypes.scala

```diff
@@ -1,9 +1,21 @@
 /*
 rewrites = ExplicitReturnTypes
+explicitReturnTypes.memberKind = [Def, Var]
+explicitReturnTypes.memberVisibility = [Public, Protected]
  */
 package test
 
 object ExplicitReturnTypes {
+  val a = 1
+  def b() = List
+  var c = true
+  protected val d = 1.0f
```

Comment:
It's good idea to add `skipSimpleDefinitions` into ExplicitReturnType option. I added.

As you mentioned, we can revisit a definition of `simple` later on :)